src/main.rs

```diff
@@ -155,6 +155,11 @@ where
                         let mut ui = ui.lock().unwrap();
                         let paused = paused.load(Ordering::SeqCst);
                         let ui_offset = ui_offset.load(Ordering::SeqCst);
+                        let interface = match opts.interface {
+                            Some(ref interface_name) => interface_name.to_string(),
+                            None => "All Interfaces".to_string(),
+                        };
```

Comment:
There is nothing wrong with this, but you can also consider using `Option::cloned` and `Option::unwrap_or`. See https://doc.rust-lang.org/std/option/enum.Option.html.